### ISSUE TITLE
Startup script needs a rework

### ISSUE THREAD

**Issue by ronoaldo** [ronoaldo/devbox#4]
The startup script currently always recreates all configuration files. It would be nice to skip them if setup was already done once.

Also, the safe_formata_and_mount command no longer exists so we need o update it to setup the home drive only once and add it to /etc/fstab.

It would be nice to include etckeeper package to keep track of config changes!

Some packages may need a rework too to the new Debian 9 stable release.

**Comment by ronoaldo** [comment]
Somewhat no longer useful as we now have better maintained options like Cloud Shell, Google Cloud Workspaces, Github Workspaces and others.
